### PR TITLE
Unset sender_string if not used

### PR DESCRIPTION
**Pull request by AntonioNigro**
If `sender_string` is empty, Skebby ignore the `sender_number` and uses the Default sender set in the Administration Panel

### CODE REVIEW

File: lib/Client/Client.php [hunk 1]
```diff
@@ -177,6 +177,14 @@ private function prepareRequest(Sms $sms)
             'charset' => urlencode($this->config['charset']),
         ];
 
+        //
+        // if sender_string is passed and is empty, it's impossible to use sender_number as sender,
+        // Skebby will use the default sender set in Skebby Administration Panel
+        //
```
Comment (alekitto) [inline lib/Client/Client.php]: Prefer `/* ... */` for multiline comments.

File: lib/Client/Client.php [hunk 2]
```diff
@@ -177,6 +177,14 @@ private function prepareRequest(Sms $sms)
...
+        if (trim($request['sender_string']) == '') {
```
Comment (alekitto) [inline lib/Client/Client.php]: Please use `===` for strict comparison.

**Review by alekitto — CHANGES_REQUESTED** [review]
Good point! And thanks for your contribution.

It seems that your modifications broke the test suite. Can you please fix that and add a test case for this scenario?
I also left some style comments

**Comment by p365labs** [comment]
@AntonioNigro any chances you can update ur PR ? tnx

**Comment by AntonioNigro** [comment]
Sorry, full of tasks! Next week i will :)

**Comment by AntonioNigro** [comment]
There is no need for a new test class, just cleaned up the test provided: the query string doesn't need an empty sender_string param. A new test class should be the same as ClientTest. Tnx

**Review by alekitto — APPROVED** [review]

**Comment by alekitto** [comment]
Thanks @AntonioNigro
